### DISCUSSION TITLE
Custom Logo

### DISCUSSION THREAD

**Discussion by juliloving** [Eloston/ungoogled-chromium#1531] [Ideas]
Hi all! In the past few days I've been working on a custom logo for Ungoogled Chromium. Naturally, I emailed Eleston about this, and they suggested posting it for a public opinion. [Here it is!](https://cubeupload.com/im/simplegenrecubiccent/Ungoogled.png) Let me know any thoughts or ideas you have :D

**Comment by networkException** [comment]
For anyone that doesn't to disable their adblock just to see the logo, here you go
![image](https://user-images.githubusercontent.com/42888162/120626146-ab04f500-c462-11eb-9c90-59c5e21e1ad3.png)

**Reply by mdedonno1337** [reply]
Too flashy for me

**Reply by mealwhiles** [reply]
In my opinion, the lego should be kept the same as regular Chromium. The point of the project is to keep the experience as close to Chromium as possible, after all.

**Reply by juliloving** [reply]
> In my opinion, the lego should be kept the same as regular Chromium. The point of the project is to keep the experience as close to Chromium as possible, after all.

Hi! I thought of this, and that's why I only made a few minor changes to the otherwise blue Chromium logo, although I thought a custom design would be a good way to represent and distinct the project as a unique fork, I incorporated both of these ideas! ^-^

**Reply by Eloston** [reply]
I think a custom logo is fine to include in packaging repos. I don't think we should include more rebranding changes in the core repo (we only have one patch right now that does mark the browser as ungoogled-chromium).

I think some packaging types, like Android, may benefit from it. @wchen342 already changed the app name from the standard one.
